dotnet/src/Functions/Functions.OpenAPI/Builders/Serialization/PipeDelimitedStyleParameterSerializer.cs

```diff
@@ -21,19 +21,16 @@ public static string Serialize(RestApiOperationParameter parameter, string argum
     {
         const string ArrayType = "array";
 
-        if (parameter is null)
-        {
-            throw new ArgumentNullException(nameof(parameter));
-        }
+        Verify.NotNull(parameter);
 
         if (parameter.Style != RestApiOperationParameterStyle.PipeDelimited)
         {
-            throw new SKException($"Unexpected Rest Api operation parameter style `{parameter.Style}`. Parameter name `{parameter.Name}`.");
+            throw new ArgumentException($"Unexpected Rest Api operation parameter style `{parameter.Style}`. Parameter name `{parameter.Name}`.", nameof(parameter));
         }
 
         if (parameter.Type != ArrayType)
         {
-            throw new SKException($"Serialization of Rest API operation parameters of type `{parameter.Type}` is not supported for the `{RestApiOperationParameterStyle.PipeDelimited}` style parameters. Parameter name `{parameter.Name}`.");
+            throw new ArgumentException($"Serialization of Rest API operation parameters of type `{parameter.Type}` is not supported for the `{RestApiOperationParameterStyle.PipeDelimited}` style parameters. Parameter name `{parameter.Name}`.", nameof(parameter));
```

Comment:
Nit: Capitalise `API` consistently, in other messages it's `Api`. I suggest we use `API` in all of the messages.

Reply:
Fixed all occurrences I could find